app/downloadshandler.py

```diff
@@ -20,9 +20,7 @@
     get_fields,
 )
 from app.searchfunctions import (
-    get_all_assignment_status,
-    get_microservice_records,
-)
+    get_microservice_records, )
```

Comment:
If you remove the brackets here, it should all fit on one line cleanly.

Reply:
removed